auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -60,173 +60,14 @@
- * <h2>Sign-out</h2>
- * <p>
- * With the integrations provided by AuthUI, signing out a user is a multi-stage process:
- * <p>
- * <ol>
- *     <li>The user must be signed out of the {@link FirebaseAuth} instance.</li>
- *     <li>Smart Lock for Passwords must be instructed to disable automatic sign-in, in
- *         order to prevent an automatic sign-in loop that prevents the user from switching
- *         accounts.
- *     </li>
- *     <li>If the current user signed in using either Google or Facebook, the user must also be
- *         signed out using the associated API for that authentication method. This typically
- *         ensures that the user will not be automatically signed-in using the current account
- *         when using that authentication method again from the authentication method picker, which
- *         would also prevent the user from switching between accounts on the same provider.
- *     </li>
- * </ol>
- * <p>
- * In order to make this process easier, AuthUI provides a simple
- * {@link AuthUI#signOut(Activity) signOut} method to encapsulate this behavior. The method returns
- * a {@link Task} which is marked completed once all necessary sign-out operations are completed:
- * <p>
- * <pre>
- * {@code
- * public void onClick(View v) {
- *   if (v.getId() == R.id.sign_out) {
- *       AuthUI.getInstance()
- *           .signOut(this)
- *           .addOnCompleteListener(new OnCompleteListener<AuthResult>() {
- *             public void onComplete(@NonNull Task<AuthResult> task) {
- *               // user is now signed out
- *               startActivity(new Intent(MyActivity.this, SignInActivity.class));
- *               finish();
- *             });
- *   }
- * }
- * </pre>
- * <p>
- * <h2>IDP Provider configuration</h2>
- * <p>
- * Interacting with identity providers typically requires some additional client configuration.
- * AuthUI currently supports Google Sign-in and Facebook Sign-in, and currently requires the
- * basic configuration for these providers to be specified via string properties:
- * <p>
- * <ul>
- * <p>
- * <li>Google Sign-in: If your app build uses the
- * <a href="https://developers.google.com/android/guides/google-services-plugin">Google
- * Services Gradle Plugin</a>, no additional configuration is required. If not, please override
- * {@code R.string.default_web_client_id} to provide your
- * <a href="https://developers.google.com/identity/sign-in/web/devconsole-project">Google OAuth
- * web client id.</a>
- * </li>
- * <p>
- * <li>Facebook Sign-in: Please override the string resource
- * {@code facebook_application_id} to provide the
- * <a href="https://developers.facebook.com/docs/apps/register">App ID</a> for your app as
- * registered on the
- * <a href="https://developers.facebook.com/apps">Facebook Developer Dashboard</a>.
- * </li>
- * <p>
- * </ul>
+ * See the <a href="https://github.com/firebase/FirebaseUI-Android/tree/master/auth#using-firebaseui-for-authentication">README</a>
```

Comment:
Let's just link to the README itself rather than a section since those headers could change:
https://github.com/firebase/FirebaseUI-Android/blob/master/auth/README.md